Microsoft.Toolkit.Graph.Controls/Providers/GraphExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Graph.Providers
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public static class GraphExtensions
```

Comment:
We could probable simplify this to just be Graph as these aren't really extensions to anything and `GraphConfig` to just `Config`?

Then it'd be:
```xml
<providers:Graph.Config>
    <providers:MockConfig/>
</providers:Graph.Config>
```

Which would be a little less verbose? Thoughts?

Reply:
Love it :)